api/src/main/java/ca/bc/gov/educ/api/dataconversion/service/student/StudentXProgramEventService.java

```diff
@@ -45,11 +46,7 @@ public <T extends Object> void processEvent(T request, Event event) {
         TraxXProgramDTO xprogram = (TraxXProgramDTO) request;
         if (xprogram != null && constants.isGradUpdateEnabled()) {
             // Get Access Token
-            ResponseObj res = restUtils.getTokenResponseObject();
-            String accessToken = null;
-            if (res != null) {
-                accessToken = res.getAccess_token();
-            }
+            String accessToken = restUtils.fetchAccessToken();
```

Comment:
If we aren't changing now, do we have a ticket for removing access token from header for this api?

Reply:
A new client is required in keycloak to utilize client credential auth.  I think a separate ticket to handle this is better.